src/components/shared/ReconfigurationOption.tsx

```diff
@@ -0,0 +1,72 @@
+import { CheckCircleFilled } from '@ant-design/icons'
+import { ThemeContext } from 'contexts/themeContext'
+import { CSSProperties, useContext } from 'react'
+
+import { Strategy } from 'constants/ballotStrategies/ballotStrategies'
+
+// This component is either the selected option which opens the drawer of all options, or is an option within that drawer.
+export default function ReconfigurationOption({
+  title,
+  content,
+  index,
+  strategy,
+  selected,
+  onSelectBallot,
+  onClick,
+}: {
+  title: string
+  content: JSX.Element
+  index: number
+  strategy: Strategy
+  selected: boolean
+  onSelectBallot?: Function
+  onClick?: Function
+}) {
+  const { colors, radii } = useContext(ThemeContext).theme
+
+  const cardStyles: CSSProperties = {
+    display: 'flex',
+    padding: '1rem',
+    borderRadius: radii.md,
+    cursor: 'pointer',
+    ...(selected
+      ? { border: '1px solid ' + colors.stroke.action.primary }
+      : {}),
+    backgroundColor: colors.background.l1,
+  }
+
+  return (
+    <div
+      key={index}
+      className="clickable-border"
+      style={cardStyles}
+      onClick={() => {
+        if (onSelectBallot) {
```

Comment:
**question**: why do we need both handlers?

Reply:
Yep lol made it a singular